config/config_test.go

```diff
@@ -268,7 +275,7 @@ func Test_Defaults(t *testing.T) {
 			expectedValue: 61090,
 		},
 	}
-	file, err := ioutil.TempFile("", "genesisConfig")
+	file, err := os.CreateTemp("", "genesisConfig")
```

Comment:
why do you think is better os.CreateTemp vs ioutil.TempFile??

Reply:
because the ioutil.TempFile is deprecated from go 1.17:
![image](https://user-images.githubusercontent.com/10800144/198227098-f20604c6-6ea7-4725-bd7c-eb82d5bf81e3.png)

:)